src/omnisharp/launcher.ts

```diff
@@ -161,17 +161,26 @@ function launch(cwd: string, args: string[]): Promise<LaunchResult> {
     return PlatformInformation.GetCurrent().then(platformInfo => {
         const options = Options.Read();
 
+        let envVars = {};
+        if (options.useEditorFormattingSettings) 
+        {
+            let editorConfig = vscode.workspace.getConfiguration('editor');
+            envVars['formattingOptions:useTabs'] = !editorConfig.get('insertSpaces', true);
+            envVars['formattingOptions:tabSize'] = editorConfig.get('tabSize', 4);
+            envVars['formattingOptions:indentationSize'] = editorConfig.get('tabSize', 4);
+        }
```

Comment:
Can't these just be passed as additional arguments? I though OmniSharp's configuration builder would pick these up from the command line.